assets/gaming/Global_Ton.json

```diff
@@ -0,0 +1,20 @@
+{
+  "metadata": {
+    "label": "global_ton",
+    "category": "gaming",
+    "subcategory": "",
+    "website": "https://t.me/Global_TON_bot ",
```

Comment:
Please remove the space from the end of the website field. 

All other stuff is great.